lib/data_recency_minutes/main.py

```diff
@@ -104,9 +104,9 @@ async def get_data_freshness(
     if most_recent_ts == 0:
         return None
 
-    data_freshness = round((utc_endtime.timestamp() - most_recent_ts) / 60)
-
-    return data_freshness
+    data_recency_minutes = round((utc_endtime.replace(tzinfo=timezone.utc).timestamp()
```

Comment:
Does adding the timezone change the resulting value over leaving timezone agnostic?

Reply:
Is this the case, or does it depend on whether the `datetime` object passed in is timezone-aware?

e.g. in a case where this is passed `datetime(..., tzinfo=tzinfo.tzname("America/Denver"))`, using `.replace(tzinfo=timezone.utc)` wouldn't cast the time to UTC and instead would just replace the timezone right?